--- exercises/003_assignment.zig	2023-10-03 22:15:22.122241138 +0200
+++ answers/003_assignment.zig	2023-10-05 20:04:06.856096469 +0200
@@ -34,12 +34,12 @@
 const std = @import("std");
 
 pub fn main() void {
-    const n: u8 = 50;
+    var n: u8 = 50;
     n = n + 5;
 
-    const pi: u8 = 314159;
+    const pi: u32 = 314159;
 
-    const negative_eleven: u8 = -11;
+    const negative_eleven: i8 = -11;
 
     // There are no errors in the next line, just explanation:
     // Perhaps you noticed before that the print function takes two
