--- exercises/045_optionals.zig	2023-10-03 22:15:22.122241138 +0200
+++ answers/045_optionals.zig	2023-10-05 20:04:07.046100027 +0200
@@ -29,7 +29,7 @@
 
     // Please threaten the result so that answer is either the
     // integer value from deepThought() OR the number 42:
-    const answer: u8 = result;
+    const answer: u8 = result orelse 42;
 
     std.debug.print("The Ultimate Answer: {}.\n", .{answer});
 }
