--- exercises/060_floats.zig	2023-10-03 22:15:22.125574535 +0200
+++ answers/060_floats.zig	2023-10-05 20:04:07.112767942 +0200
@@ -43,7 +43,7 @@
     //
     // We'll convert this weight from tons to kilograms at a
     // conversion of 907.18kg to the ton.
-    const shuttle_weight: f16 = 907.18 * 2200;
+    const shuttle_weight: f32 = 907.18 * 2200.0;
 
     // By default, float values are formatted in scientific
     // notation. Try experimenting with '{d}' and '{d:.3}' to see
