--- exercises/060_floats.zig	2023-11-06 19:45:03.609687304 +0100
+++ answers/060_floats.zig	2023-11-06 19:44:49.249419994 +0100
@@ -43,7 +43,7 @@
     //
     // We'll convert this weight from pound to kilograms at a
     // conversion of 0.453592kg to the pound.
-    const shuttle_weight: f16 = 0.453592 * 4480e6;
+    const shuttle_weight: f32 = 0.453592 * 4.480e6;
 
     // By default, float values are formatted in scientific
     // notation. Try experimenting with '{d}' and '{d:.3}' to see
