--- exercises/066_comptime.zig	2023-10-03 22:15:22.125574535 +0200
+++ answers/066_comptime.zig	2023-10-05 20:04:07.139435109 +0200
@@ -62,8 +62,8 @@
     // types with specific sizes. The comptime numbers will be
     // coerced (if they'll fit!) into your chosen runtime types.
     // For this it is necessary to specify a size, e.g. 32 bit.
-    var var_int = 12345;
-    var var_float = 987.654;
+    var var_int: u32 = 12345;
+    var var_float: f32 = 987.654;
 
     // We can change what is stored at the areas set aside for
     // "var_int" and "var_float" in the running compiled program.
