--- exercises/069_comptime4.zig	2023-10-03 22:15:22.125574535 +0200
+++ answers/069_comptime4.zig	2023-10-05 20:04:07.152768692 +0200
@@ -42,8 +42,8 @@
 //     2) Sets the size of the array of type T (which is the
 //        sequence we're creating and returning).
 //
-fn makeSequence(comptime T: type, ??? size: usize) [???]T {
-    var sequence: [???]T = undefined;
+fn makeSequence(comptime T: type, comptime size: usize) [size]T {
+    var sequence: [size]T = undefined;
     var i: usize = 0;
 
     while (i < size) : (i += 1) {
