--- exercises/072_comptime7.zig	2023-10-03 22:15:22.125574535 +0200
+++ answers/072_comptime7.zig	2023-10-05 20:04:07.169435669 +0200
@@ -35,7 +35,7 @@
     // at compile time.
     //
     // Please fix this to loop once per "instruction":
-    ??? (i < instructions.len) : (???) {
+    inline while (i < instructions.len) : (i += 3) {
 
         // This gets the digit from the "instruction". Can you
         // figure out why we subtract '0' from it?
