--- exercises/075_quiz8.zig	2023-11-21 14:48:15.440702720 +0100
+++ answers/075_quiz8.zig	2023-11-21 14:50:23.453311616 +0100
@@ -49,7 +49,11 @@
 //
 // Please fill in the body of this function!
 fn makePath(from: *Place, to: *Place, dist: u8) Path {
-
+    return Path{
+        .from = from,
+        .to = to,
+        .dist = dist,
+    };
 }
 
 // Using our new function, these path definitions take up considerably less
