--- exercises/090_async7.zig	2023-10-03 22:15:22.125574535 +0200
+++ answers/090_async7.zig	2023-10-05 20:04:07.249437167 +0200
@@ -29,7 +29,7 @@
     // The main() function can not be async. But we know
     // getBeef() will not suspend with this particular
     // invocation. Please make this okay:
-    var my_beef = getBeef(0);
+    var my_beef = nosuspend getBeef(0);
 
     print("beef? {X}!\n", .{my_beef});
 }
