--- exercises/099_formatting.zig	2024-11-07 21:45:10.459123650 +0100
+++ answers/099_formatting.zig	2024-11-07 21:43:55.154345991 +0100
@@ -131,7 +131,7 @@
         for (0..size) |b| {
             // What formatting is needed here to make our columns
             // nice and straight?
-            print("{???} ", .{(a + 1) * (b + 1)});
+            print("{d:>3} ", .{(a + 1) * (b + 1)});
         }
 
         // After each row we use double line feed:
