--- exercises/105_threading2.zig	2024-03-23 16:35:14.754540802 +0100
+++ answers/105_threading2.zig	2024-03-23 16:38:00.577539733 +0100
@@ -81,8 +81,8 @@
         defer handle1.join();
 
         // Second thread to calculate the minus numbers.
-        ???
-        
+        const handle2 = try std.Thread.spawn(.{}, thread_pi, .{ &pi_minus, 3, count });
+        defer handle2.join();
     }
     // Here we add up the results.
     std.debug.print("PI ≈ {d:.8}\n", .{4 + pi_plus - pi_minus});
